docs/source/contributing/development.rst

```diff
@@ -47,9 +47,36 @@ For first-time contributors
    - ``upstream`` the ManimCommunity repository
 
 #. Install Manim:
- 
-   - See our :doc:`instructions for developers<../installation/for_dev>` for
-     details and continue here afterwards.
+
+   - Follow the steps in our :doc:`installation instructions 
+     <../installation>` to install **Manim's dependencies**,
+     primarily ``ffmpeg`` and ``LaTeX``.
+   
+   - We recommend `Poetry <https://python-poetry.org>`__ to manage your
+     developer installation of Manim. Poetry is a tool for dependency
+     management and packaging in Python. It allows to declare the libraries
+     your project depends on and it will manage (install / update) them 
```

Comment:
```suggestion
     your project depends on, and it will manage (install / update) them 
```
This sentence is still a little awkward with this change.